tests/unit/debug/deprecate-test.js

```diff
@@ -218,13 +219,13 @@ URL    https://example.com`,
         url: 'https://example.com',
       });
     }).to.throw(
-      'The API deprecated by ember-cli.foo-method was removed in ember-cli 3.0.0. The message was: The `foo` method is deprecated. Please see https://example.com for more details.'
+      'The API deprecated by ember-cli.foo-method was removed in ember-cli@3.0.0. The message was: The `foo` method is deprecated. Please see https://example.com for more details.'
     );
   });
 
   it('throws an deprecation if the pre-release ember-cli version is greater than the until version of deprecation', function () {
-    const OVERRIDE_VERSION = process.env.OVERRIDE_DEPRECATION_VERSION;
-    process.env.OVERRIDE_DEPRECATION_VERSION = '9.0.0-beta.1';
+    // this shadows the imported deprecate function on purpose because we need to be able to override the version
+    let deprecate = makeDeprecate('ember-cli', '9.0.0-beta.1');
```

Comment:
Not specific to this test but this removes a key feature of deprecation testing. By having a CI job that simulates all deprecations as having past their 'until' we test that deprecations have been implemented in a way that works before and after the deprecation begins to throw:

https://github.com/ember-cli/ember-cli/blob/master/.github/workflows/ci.yml#L123-L133

This test was the way to guard to ensure `OVERRIDE_DEPRECATION_VERSION` was working

Reply:
Any thoughts on how we can test OVERRIDE_DEPRECATION_VERSION? In the original we punted and put this test in so we'd at least see it and wonder.

Reply:
And this one^